MdeModulePkg/Universal/CapsulePei/Common/CapsuleCoalesce.c

```diff
@@ -333,7 +333,7 @@ ValidateCapsuleIntegrity (
     return NULL;
   }
 
-  DEBUG ((EFI_D_INFO, "Ptr - 0x%lx\n", (UINT64)Ptr)); //MS_CHANGE uint64 cast
+  DEBUG ((EFI_D_INFO, "Ptr - 0x%lx\n", (UINTN)Ptr)); //MS_CHANGE uintn cast

```

Comment:
i don't think this is right.  The whole idea is the value will be printed as 64bit value (long) so it should be casted to UINT64 in all cases.